packages/launch/src/launch/index.tsx

```diff
@@ -48,10 +61,10 @@ const FocusedLaunchModal: React.FunctionComponent< Props > = ( {
 				className={ classNames( 'launch__focused-modal', {
 					'launch__focused-modal--hide-title': ! isModalTitleVisible,
 				} ) }
+				bodyOpenClassName="has-focused-launch-modal"
 				overlayClassName={ classNames( 'launch__focused-modal-overlay', {
-					'launch__focused-modal-overlay--delay-animation-in': shouldDisplaySuccessView,
+					'launch__focused-modal-overlay--delay-animation-in': isLaunchImmediately,
```

Comment:
Context: As to why it has 1 sec delay, easier on the eyes. See: https://github.com/Automattic/wp-calypso/pull/47808#issuecomment-734625847